src/index.html

```diff
@@ -6,21 +6,428 @@
     <title>EasyHouse</title>
     <link rel="stylesheet" href="style.css">
 </head>
-<body>
+<body class="body_container">
     <h1>Easyhouse</h1>
-    <header>
+    <header class="project_menu">
```

Comment:
I noticed that you introduced a project_menu class for styling purposes. However, it appears that this class duplicates the styles already applied to the header element. Consider removing the project_menu class to eliminate redundancy in your styles.